src/main/java/net/lingala/zip4j/util/FileUtils.java

```diff
@@ -181,41 +181,46 @@ public static String getZipFileNameWithoutExtension(String zipFile) throws ZipEx
     return splitZipFiles;
   }
 
-  public static String getRelativeFileName(String file, String rootFolderPath) {
+  public static String getRelativeFileName(String file, String rootFolderPath) throws ZipException {
```

Comment:
Same comment as above also here: Is it necessary to also change this method? I think just changing it above in `AbstractExtractFileTask.java` is enough?

Reply:
This is necessary. Please refer to my following comment.